config/semi_automated.yml

```diff
@@ -67,7 +67,7 @@ parameters:
             regex: '\|HotCat\]\]|Gadget-Hotcat(?:check)?\.js\|Script|\]\] via HotCat|\[\[WP:HC\|'
             link: Special:MyLanguage/Project:HotCat
         Cat-a-lot:
-            regex: '\[\[Help:Cat-a-lot\|Cat-a-lot\]\]'
+            regex: '\[\[Help:Cat-a-lot\|Cat-a-lot\]\]|\(CatAlot\)|\[\[c:Help:Cat-a-lot\|Cat-a-lot\]\]'
```

Comment:
I think we can get away with just
```
Help:Cat-a-lot\|Cat-a-lot\]\]|\(CatAlot\)
```
The basic idea is to make the regex as short yet simple as possible, while preventing false positives. Here it seems unlikely someone would manually put `Help:Cat-a-lot|Cat-a-lot]]` in their edit summary.

Reply:
done